### ISSUE TITLE
Missing Logo Image for Hyperledger Fabric on Landing Page

### ISSUE THREAD

**Issue by PrathamSikka24** [hyperledger/caliper#1531]
### Which Caliper version are you using?

latest

### Which Node.JS version are you using?

v18.19.1

### Which operating system are you using?

Ubuntu 22.04.3 LTS(through WSL)

### Please provide some context for your error. For example, when did the error occur? What were you trying to achieve, and how?

I was visiting the [Hyperledger Caliper documentation website ](https://hyperledger.github.io/caliper/) with the intent to learn more about the supported blockchains. On the landing page, I observed that the logo image for Hyperledger Fabric was not displayed, unlike the logos for other blockchain frameworks such as Hyperledger Besu, Ethereum, and FISCO BCOS.

![Screenshot (321)](https://github.com/hyperledger/caliper/assets/116445216/10c81fe0-efc2-47fa-aad3-90e2099c89e0)

### What was the observed incorrect behavior?

The Hyperledger Fabric logo does not appear on the landing page. The expected logo image is replaced with alt text reading "Fabric".

### Please provide the error logs and their surroundings.

```text
N/A - This issue relates to the website's content display rather than an error log from the Caliper tool itself.
```

### Please provide your benchmark configuration file content, if possible.

```yaml
N/A - This issue is not related to the benchmark configuration.
```

### Please provide your network configuration file content, if possible.

```yaml
N/A - This issue is not related to the network configuration.
```

### Please provide your workload module content, if possible.

```nodejs
N/A - This issue is not related to a specific workload module.
```

**Comment by PrathamSikka24** [comment]
I have identified the missing Hyperledger Fabric logo on the landing page and have successfully added the correct image file to the website's assets. I ensured the logo's path corresponds to the expected location. The logo now displays as intended alongside the other project logos.

To address this issue, I have committed the fix to my repository fork. I have tested the updated image path in my local environment, and the logo is now visible.

The commit with the fix can be found in my fork: [link to the commit](https://github.com/PrathamSikka24/caliper/commit/bf053fdfa3d89d914cdf2024dd328135a0f75ede)
